modules/compliance-supported-profiles.adoc

```diff
@@ -8,60 +8,72 @@
 The Compliance Operator provides the following compliance profiles:
 
 .Supported compliance profiles
-[cols="40%,50%,10%", options="header"]
+[cols="10%,40%,10%,40%", options="header"]
 
 |===
-|Profile name
+|Profile
 |Profile title
 |Compliance Operator version 
+|Compliance benchmark
 
 |ocp4-cis
-|link:https://www.cisecurity.org/cis-benchmarks/[CIS Red Hat OpenShift Container Platform 4 Benchmark]
+|CIS Red Hat OpenShift Container Platform 4 Benchmark
 |0.1.39+
+|link:https://www.cisecurity.org/cis-benchmarks/[CIS Benchmarks &#8482;] footnoteref:[fn1,To locate the CIS RedHat OpenShift Container Platform v4 Benchmark, go to  link:https://www.cisecurity.org/cis-benchmarks/[CIS Benchmarks] and type `Kubernetes` in the search box. Click on Kubernetes > Download Latest CIS Benchmark, where you can then register to download the benchmark.]
```

Comment:
I think "Kubernetes > Download Latest CIS Benchmark" should be bold (https://www.ibm.com/docs/en/ibm-style?topic=elements-ui). If this doesn't fit the referenced example, I'd reword to "Click on **Kubernetes** and then **Download Latest CIS Benchmark**."

Reply:
Done